README.md

```diff
@@ -40,18 +40,18 @@ To install the MapboxNavigation framework in an application using [Swift Package
      login mapbox
      password PRIVATE_MAPBOX_API_TOKEN
    ```
-   where _PRIVATE_MAPBOX_API_TOKEN_ is your Mapbox API token with the `DOWNLOADS:READ` scope. 
+   where _PRIVATE_MAPBOX_API_TOKEN_ is your Mapbox API token with the `DOWNLOADS:READ` scope.
 
 1. In Xcode, go to File ‣ Swift Packages ‣ Add Package Dependency.
 
 1. Enter `https://github.com/mapbox/mapbox-navigation-ios.git` as the package repository and click Next.
 
-1. Set Rules to Version, Up to Next Major, and enter `2.0.0-alpha.1` as the minimum version requirement. Click Next.
+1. Set Rules to Version, Up to Next Major, and enter `2.0.0-beta.25` as the minimum version requirement. Click Next.
```

Comment:
Should this line be changed? As I'm not sure that with `2.0.0-alpha.1` Xcode can update to the very latest beta.

Reply:
Yes, why not.